Update the WCF Web service reference guide (#15511)
dotnet/docs: docs/core/additional-tools/wcf-web-service-reference-guide.md

## Next steps

> ### See also

Reviewer: I'd recommend ## See also and removing ## Next steps. Also I think Feedback & questions and Release notes can be H2s.

Author: Fixed

> - [Get started with Windows Communication Foundation applications](https://docs.microsoft.com/en-us/dotnet/framework/wcf/getting-started-tutorial)

Reviewer: ```suggestion
- [Get started with Windows Communication Foundation applications](../../../framework/wcf/getting-started-tutorial.md)
```

Author: @gewarren I took your suggestion, but on the markdown diff the relative links don't seem to work as expected. Is there any way to verify this is pointing to the right page? If the current document is in "docs/core/additional-tools" and the document we're linking is in "docs/framework/wcf shouldn't the link be ../../framework/wcf"?

Reviewer 2: @dasetser, I think you're correct. It should `../../` not `../../../`. You can push a commit and see if it will redirect you correctly on GitHub diff.

Author: @Youssef1313 That does look to have fixed that link. Thanks!

> - [Windows Communication Foundation Services and WCF data services in Visual Studio](https://docs.microsoft.com/en-us/visualstudio/data-tools/windows-communication-foundation-services-and-wcf-data-services-in-visual-studio)

Reviewer: ```suggestion
- [Windows Communication Foundation services and WCF data services in Visual Studio](/visualstudio/data-tools/windows-communication-foundation-services-and-wcf-data-services-in-visual-studio)
```

So the link works in offline books.

> - [WCF Supported features on .NET Core](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v2.1.0.md)

Reviewer: ```suggestion
- [WCF supported features on .NET Core](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v2.1.0.md)
```

### Feedback & questions

> If you have any questions or feedback, report it at [Visual Studio's developer community](https://developercommunity.visualstudio.com/).

Reviewer: ```suggestion
If you have any questions or feedback, report it at [Developer Community](https://developercommunity.visualstudio.com/) using the [Report a problem](/visualstudio/ide/how-to-report-a-problem-with-visual-studio) tool.
```
